Vegeting/Screens/ChatRoom/Cell/InterestCollectionViewCell.swift

```diff
@@ -26,6 +26,8 @@ class InterestCollectionViewCell: UICollectionViewCell {
         return label
     }()
     
+    var isSelectedCell = false
+    
```

Comment:
p2;
데이크 예전에 말씀드렸던 CollectionViewCell에 내장된 override변수인 isSelected를 소개시켜 드릴게요!
해당 변수는 collectionView를 위해서 애플이 만들어 둔 기능으로서, 선택될 시 해당 변수의 값이 바뀌게 됩니다.!
```suggestion
override var isSelected: Bool {
        didSet {
            applySelectedState()
        }
    }
```

Reply:
https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/143/commits/f25642e9e854c1afa40d2ecf8461b3c42ed4e218#diff-621530c17340448afe600a661e652bcb33b97219af1cb0be72054c53c0783a11R29-R33에서 반영하였습니다.